project-base/translations/messages.en.po

```diff
@@ -175,6 +175,9 @@ msgstr ""
 msgid "Email"
 msgstr ""
 
+msgid "Email was already subscribed"
+msgstr ""
```

Comment:
this is no longer used ;)

Reply:
3aa7b9914e9355409269d5cd536a1244a17c1f31